hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JPAActionManagement.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.hawkbit.repository.RepositoryProperties;
+import org.eclipse.hawkbit.repository.model.Action;
+import org.springframework.data.domain.PageRequest;
+
+/**
+ * Implements utility methods for managing {@link Action}s
+ */
+public class JPAActionManagement {
```

Comment:
can we rename it to JpaActionManagement (camel-case) to make it consistent with the other classes (e.g. JpaDeploymentManagement)?

Reply:
sure, will update it